Libplanet.Explorer/Controllers/ExplorerController.cs

```diff
@@ -36,6 +39,129 @@ public BlockChain<T> GetBlockChain()
             return chain;
         }
 
+        public class BlockType : ObjectGraphType<Block<T>>
```

Comment:
Could you move these object graph types to a separated namespace, like `Libplanet.Explorer.GraphTypes`?